lookout-style-analyzer/templates/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "lookout-style-analyzer.fullname" . }}
+  labels:
+    app: {{ template "lookout-style-analyzer.name" . }}
+    chart: {{ template "lookout-style-analyzer.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "lookout-style-analyzer.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "lookout-style-analyzer.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+        - name: data
+          {{- if .Values.app.volume.hostPath }}
+          hostPath:
+            path: {{ .Values.app.volume.hostPath }}
+          {{- else }}
+          persistentVolumeClaim:
+            claimName: {{ template "lookout-style-analyzer.fullname" . }}
+          {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - run
+            - lookout.style.format
+            - -s
+            - 0.0.0.0:{{ .Values.app.port }}
```

Comment:
This have to be a parameter - it's an address of Lookout server https://github.com/src-d/style-analyzer/blob/master/lookout/core/cmdline.py#L152 and should be configured on deployment to point to `ipv4://lookoutr: 10301` (from https://github.com/src-d/charts/pull/63/files#diff-d50f3a3496ea3f7f7965b6abb4f07ca2R31)

Reply:
It is also possible to specify a configuration file: https://github.com/src-d/style-analyzer/blob/master/lookout/core/doc/getting_started.md#environment

```
analyzer run lookout.style.format -c config.yml
```

Reply:
what's the difference between `--server` and `--request-server`?